inc/cli/class-migrate-command.php

```diff
@@ -259,7 +259,7 @@ function ppa( $args, $assoc_args ) : void {
 				// Usually invalid taxonomy, lets catch and report this.
 				if ( is_wp_error( $ppa_terms ) ) {
 					WP_CLI::error( 'There was an error fetching the PublishPress Author data, is the plugin activated?', false );
-					WP_CLI::error( $ppa_terms );
+					WP_CLI::error( $ppa_terms, false );
 					exit;
```

Comment:
`exit` alone will not return an error code whereas `WP_CLI::error()` calls `exit( 1 )` if the 2nd param is true
```suggestion
					exit( 1 );
```